content/en/continuous_integration/tests/javascript.md

```diff
@@ -43,6 +43,7 @@ The instrumentation works at runtime, so any transpilers such as TypeScript, Web
 * Mocha >= 5.2.0
   * From `dd-trace>=3.3.0`.
 
+Additionally, test suite level visibility is only supported in Agentless mode.
```

Comment:
I think we should start this section with this sentence, minus the "Additionally"